src/Business/Executor/RouteExecutor.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * This file is part of the Micro framework package.
+ *
+ * (c) Stanislau Komar <kost@micro-php.net>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Micro\Plugin\Http\Business\Executor;
+
+use Micro\Component\DependencyInjection\Autowire\AutowireHelperInterface;
+use Micro\Component\DependencyInjection\ContainerRegistryInterface;
+use Micro\Plugin\Http\Business\Matcher\UrlMatcherInterface;
+use Micro\Plugin\Http\Exception\HttpException;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+
+/**
+ * @author Stanislau Komar <kost@micro-php.net>
+ */
+readonly class RouteExecutor implements RouteExecutorInterface
+{
+    /**
+     * @param UrlMatcherInterface $urlMatcher
+     * @param ContainerRegistryInterface $containerRegistry
+     * @param AutowireHelperInterface $autowireHelper
+     */
+    public function __construct(
+        private UrlMatcherInterface $urlMatcher,
+        private ContainerRegistryInterface $containerRegistry,
+        private AutowireHelperInterface $autowireHelper,
+    )
+    {
```

Comment:
@Asisyas, please run php-cs-fixer, here `{` should be on the same line as `)`.

Reply:
fixed.